accepted/PSR-6-cache-meta.md

```diff
@@ -303,3 +303,43 @@ _**Note:** Order descending chronologically._
 [1]: https://docs.google.com/spreadsheet/ccc?key=0Ak2JdGialLildEM2UjlOdnA4ekg3R1Bfeng5eGlZc1E#gid=0
 [2]: https://docs.google.com/spreadsheet/ccc?key=0AsMrMKNHL1uGdDdVd2llN1kxczZQejZaa3JHcXA3b0E#gid=0
 [3]: https://docs.google.com/spreadsheet/ccc?key=0AsMrMKNHL1uGdEE3SU8zclNtdTNobWxpZnFyR0llSXc#gid=1
+
+8. Errata
+----------
+
+### 8.1 Handling of incorrect DateTime values in expiresAt()
+
+The `CacheItemInterface::expiresAt()` method's `$expiration` parameter is untyped
+in the interface, but in the docblock is specified as `\DateTimeInterface`.  The
+intent is that either a `\DateTime` or `\DateTimeImmutable` object is allowed.
+However, `\DateTimeInterface` and `\DateTimeImmutable` were added in PHP 5.5, and
+the authors chose not to impose a hard syntactic requirement for PHP 5.5 on the
+specification.
+
+Despite that, implementers MUST accept only `\DateTimeInterface` or compatible types
+(such as `\DateTime` and `\DateTimeImmutable`) as if the method was explicitly typed.
+(Note that the variance rules for a typed parameter may vary between language versions.)
+
+Simulating a failed type check unfortunately varies between PHP versions and thus is not
+recommended.  Instead, implementors SHOULD throw an instance of `\Psr\Cache\InvalidArgumentException`.  
+The following sample code is recommended in order to enforce the type check on the expiresAt()
+method:
+
+```php
+
+class ExpiresAtInvalidParameterException implements Psr\Cache\InvalidArgumentException {}
+
+// ...
+
+if (! (
+        null === $expiration
+        || $expiration instanceof \DateTime
```

Comment:
Unless the goal is to support PHP < 5.5, this line is redundant given the Interface check below.  If the goal is to support < 5.5, then ignore this comment.

Reply:
Support for PHP < 5.5 is the whole reason we're in this mess in the first place. :smile: